cmd/influx/write.go

```diff
@@ -152,11 +184,27 @@ func fluxWriteF(cmd *cobra.Command, args []string) error {
 			InsecureSkipVerify: flags.skipVerify,
 		},
 	}
-
 	ctx = signals.WithStandardSignals(ctx)
 	if err := s.Write(ctx, orgID, bucketID, r); err != nil && err != context.Canceled {
 		return fmt.Errorf("failed to write data: %v", err)
 	}
 
 	return nil
 }
+
+func fluxWriteDryrunF(cmd *cobra.Command, args []string) error {
```

Comment:
excellent, CLI is looking good from my end. Thanks for running with feedback here 👍